channel-policies.md

```diff
@@ -15,7 +15,7 @@ Here you can find postings for development jobs in the Triangle area
 
 Rules:
 
-1. Do not send unsolicited DMs to members about job openings
+1. Do not send unsolicited DMs or emails to members about job openings
```

Comment:
```suggestion
1. Do not send unsolicited DMs or emails to members about job openings or other solicitations
```

Might this be better to also cover these other spam messages you received?
> So far I have gotten spam for:
> - “We have jobs, come apply”
> - “We have contractors, come hire them.”
> - “We have Product XXX, come buy it.”

Are there any legitimate solicitations by DM we'd unintentionally ruling against?

Reply:
Maybe adding "commercial solicitations"?  

I could see something like "Hey, you just posted a Scala job opening and I run the Triangle Scala Whatever Group.  I just wanted to let you know that we have a monthly meetup of Scala engineers and at the end of each meeting we invite hiring companies to share their relevant job openings."

Something like that would be welcomed.

Reply:
👍 Updated.